Assignment3/cponceph-x150426/Exercise5.txt

```diff
@@ -0,0 +1,8 @@
+PREFIX db: <http://dbpedia.org/ontology/>
+
+SELECT distinct ?prop (COUNT(?c) AS ?count) WHERE {
```

Comment:
It should be count (Distinct ?c), but accepted anyway

Reply:
Well, I prefer not to accept it, because I have just realised that this query cannot be evaluated. Have you actually run it?